homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -195,7 +201,17 @@ def device_state_attributes(self):
                 # Not working --> 'Cleaning mode':
                 #    STATE_ON if self.vacuum_state.in_cleaning else STATE_OFF,
                 ATTR_CLEANING_TIME: str(self.vacuum_state.clean_time),
-                ATTR_CLEANED_AREA: round(self.vacuum_state.clean_area, 2)})
+                ATTR_CLEANED_AREA: round(self.vacuum_state.clean_area, 2),
+                ATTR_CLEANING_COUNT: str(self.clean_history.count),
+                ATTR_CLEANED_TOTAL_AREA: round(
+                    self.clean_history.total_area, 2),
+                ATTR_CLEANING_TOTAL_TIME: str(
+                    self.clean_history.total_duration),
+                ATTR_MAIN_BRUSH_LEFT: str(
+                    self.consumable_state.main_brush_left),
+                ATTR_SIDE_BRUSH_LEFT: str(
+                    self.consumable_state.side_brush_left),
+                ATTR_FILTER_LEFT: str(self.consumable_state.filter_left)})
```

Comment:
do this value change every call?

Reply:
if the vacuum is running, this values change every time. Because the more he works, the less u can use the brushes and filter.